Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/PlistDetailsSectionTranslator.swift

```diff
@@ -28,7 +28,7 @@ struct PlistDetailsSectionTranslator: RenderSectionTranslator, Decodable {
     }
     
     func translateSection(for symbol: Symbol, renderNode: inout RenderNode, renderNodeTranslator: inout RenderNodeTranslator) -> VariantCollection<CodableContentSection?>? {
-        guard let mixinVariant = symbol.mixinsVariants.allValues.first(where: { mixin in
+        guard let _ = symbol.mixinsVariants.allValues.first(where: { mixin in
```

Comment:
FIY: This entire check is redundant since the code below uses `mapFirst` of he same mixin variants. 

Because this was causing a warning in the code I removed the entire check in [this commit in another PR](https://github.com/swiftlang/swift-docc/pull/955/commits/82d63245335f14a7795c56e3ce430a7c2db0e4bc)